src/app/person/person.component.scss

```diff
@@ -68,7 +68,7 @@
       ngx-gallery-action i {
         width: 40px;
         height: 40px;
-        background-image: url("assets/images/close-icon.png");
+        background-image: url("../../assets/images/close-icon.png");
```

Comment:
this path will broke the images on staging, and I guess on prod as well. Leave this paths as is